ndmg/stats/qa_skullstrip.py

```diff
@@ -0,0 +1,244 @@
+#!/usr/bin/env python
+
+# Copyright 2016 NeuroData (http://neurodata.io)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+# Created by Vikram Chandrashekhar.
+# Edited by Greg Kiar, Eric Bridgeford and Chuankai Luo.
+# Email: Greg Kiar @ gkiar@jhu.edu   cluo16@jhu.edu
+
+import warnings
+
+warnings.simplefilter("ignore")
+import os
+import re
+import sys
+import numpy as np
+import nibabel as nb
+# import ndmg.utils as mgu
+from argparse import ArgumentParser
+from scipy import ndimage
+from matplotlib.colors import LinearSegmentedColormap
+import matplotlib as mpl
+
+mpl.use("Agg")  # very important above pyplot import
+# from nilearn.plotting.edge_detect import _edge_map as edge_map
+import matplotlib.pyplot as plt
```

Comment:
what was the deal with this? e.g., what error caused you to need to do this?

Reply:
there was something wrong with my local computer because of some environment problem.....now I fixed it

Reply:
![image](https://user-images.githubusercontent.com/55203069/70834852-68acab80-1dc9-11ea-95af-a63c9422c142.png)